checksum.go

```diff
@@ -67,7 +75,7 @@ func (c *FileChecksum) checksum(source string) error {
 			Hash:     c.Hash,
 			Actual:   actual,
 			Expected: c.Value,
-			File:     source,
+			File:     file,
```

Comment:
This would return a closed file. Is that intended?

Reply:
Oh, I misread, file isn't the file it's the path.

Reply:
Would it be better to rename the variable then?

Reply:
I agree, I think that should probably be `filePath` !